demo/full/styles/style.css

```diff
@@ -45,25 +50,48 @@ body {
 }
 
 .choice-inputs-wrapper {
-  margin: 10px auto 30px auto;
+  margin: 10px auto 5px auto;
+  padding: 0 0 5px 0;
 }
 
 .content-inputs {
   display: flex;
-  justify-content: flex-start;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.content-inputs > * {
+  display: flex;
+  justify-content: space-between;
+  align-items: baseline;
 }
 
 .choice-input {
   margin: 0px;
   display: inline;
 }
 
+.choice-input > select {
+  padding: 1px;
+  display: flex;
+  background-color: white;
+}
+
 .choice-input.text-input {
-  width: 80%;
-  margin-right: 10px;
-  margin-bottom: 10px;
+  font-family: Roboto;
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Unexpected missing generic font family ([css:S4649](https://sonarcloud.io/organizations/rx-player/rules#rule_key=css%3AS4649))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=rx-player&pullRequest=312&issues=AWXXLiJtashydt7kNvQD&open=AWXXLiJtashydt7kNvQD)

[ISSUE_KEY]: AWXXLiJtashydt7kNvQD